payments-core/src/main/java/com/stripe/android/GooglePayJsonFactory.kt

```diff
@@ -464,7 +471,11 @@ class GooglePayJsonFactory constructor(
         internal val merchantName: String? = null
     ) : Parcelable
 
-    private companion object {
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    companion object {
+        @JvmStatic
+        var cardBrandFilter: CardBrandFilter = DefaultCardBrandFilter
```

Comment:
Agree that is the optimal approach, however I think that would require public API changes to this class which I don't think is desirable for this use-case, since we'd need to expose a CardBrandFilter API for just Google Pay as well.